pkg/minikube/openshiftversions/openshift_versions.go

```diff
@@ -42,7 +44,11 @@ func PrintOpenShiftVersions(output io.Writer) {
 	fmt.Fprint(output, "The following OpenShift versions are available: \n")
 
 	for _, version := range versions {
-		fmt.Fprintf(output, "\t- %s\n", *version.TagName)
+		v, _ := semver.Parse(*version.TagName)
+		rang, _ := semver.ParseRange(fmt.Sprintf(">%s", constants.SupportedOpenshiftTag))
```

Comment:
_rang_? I guess just 'r' would be better. Or a different name altogether. I guess you drop the 'e' due to name conflict?